app/Controllers/Http/Dashboard/LoginController.ts

```diff
@@ -40,12 +42,38 @@ export default class LoginController {
     try {
       const { mail, password } = request.all();
 
+      // Check if user with email exists
+      const user = await User.query().where('email', mail).first();
+      if (!user?.email) {
+        return response.status(401).send({
+          message: 'User credentials not valid (Invalid mail)',
```

Comment:
Get rid of the "(Invalid mail)" part which can indicate that it's the email which is incorrect and not the pw.